index.html

```diff
@@ -11,6 +11,7 @@
     <div class="detail">
       <div class="detail__cover">
         <img src="./assets/book-cover.png" alt="cover of the book" class="detail__cover__img" />
+        <img src="./assets/badge.png" alt="badge of the book" class="detail__cover__badge" />
```

Comment:
Remember to use as less html code as possible for clarity and reusability. In this case, try just to use just the container element and apply the image as a decorator using only CSS (hint: you can use the ::before selector), that way you will have a single reusable element (the container) that contains the main image, and manage the decorative part via CSS only. If you have any doubt about this ask me.

Reply:
Done 😄